### DISCUSSION TITLE
Attribute expansion not working on 1.21.3

### DISCUSSION THREAD

**Discussion by TheBeast572** [PlaceholderAPI/PlaceholderAPI#1091] [(2) Support] [answered]
![image](https://github.com/user-attachments/assets/0451f2aa-0dea-4a6e-b78f-f9923884cb7e)
I am running a server on 1.21.3 and tried grabbing the Attributes expansion but I keep getting this error in console.

**Comment by Andre601** [comment, marked as answer]
Report this issue to the AttributeExpansion repository: https://github.com/PlaceholderAPI/Attribute-Expansion

It's easier to get your issue fixed that way.
